app/jobs/regular/merge_user.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+
+module Jobs
+  class MergeUser < ::Jobs::Base
+
+    def execute(args)
+      target_user_id = args[:target_user_id]
+      current_user_id = args[:current_user_id]
+
+      user = User.find_by(id: args[:user_id])
+      target_user = User.find_by(id: args[:target_user_id])
+      current_user = User.find_by(id: args[:current_user_id])
+      guardian = Guardian.new(current_user)
+      serializer_opts = { root: false, scope: guardian }
+
+      if user = UserMerger.new(user, target_user, current_user).merge!
+        sleep(10)
```

Comment:
I assume this line was also used for testing?

Reply:
oops! yes.